api/api/openapi.bundle.yaml

```diff
@@ -608,6 +608,12 @@ paths:
         schema:
           format: int32
           type: integer
+      - description: boolean indicating whether to deploy the new version once updated
+        in: query
+        name: deploy
+        required: true
+        schema:
+          type: boolean
```

Comment:
Added a new flag to be included in the query string `"?deploy=(true|false)"` for the endpoint `"/projects/{project_id}/routers/{router_id}"`, which is only required for `PUT` operations, i.e. updating routers with/without deployment

Reply:
It's not your fault, but this still doesn't satisfy the criteria of `PUT` being an idempotent operation. So I would rather standardize this `create/create and deploy` operation under `POST /projects/{project_id}/routers/{router_id}/versions`

Reply:
Hmm, yeah, it's a little awkward for the same endpoint to have somewhat different operations depending on the query string; makes it hard to place it in either group.

But just to clarify, if we're moving to both `create/create and deploy` operations to `POST /projects/{project_id}/routers/{router_id}/versions`, it means we're letting the `router_versions` API handle them, as opposed to the `routers` API previously?